app/templates/views/dashboard/review-email-list.html

```diff
@@ -47,7 +47,7 @@ <h2 class="heading-medium mt-8">{{ _('Problem emails over the past 24 hours') }}
             {% for job in jobs %}
                 {% if job["bounce_count"] > 0 %}
                     <li>
-                        <a href="{{ url_for('.view_job', service_id=current_service.id, job_id=job.id, status='failed') }}">
+                        <a href="{{ url_for('.view_job', service_id=current_service.id, job_id=job.id, status='failed', pe_filter='true') }}">
```

Comment:
```suggestion
                        <a href="{{ url_for('.view_job', service_id=current_service.id, job_id=job.id, status='permanent-failure', pe_filter='true') }}">
```

If we want to just show problem addresses, I think this needs to be "`permanent-failure"`

Reply:
💯  Great catch, thank you!